packages/safe-tools-client/public/mockServiceWorker.js

```diff
@@ -2,122 +2,151 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (0.40.2).
+ * Mock Service Worker (0.48.1).
  * @see https://github.com/mswjs/msw
  * - Please do NOT modify this file.
```

Comment:
Err…

Reply:
I guess nothing has changed in this worker script since the previous version. There's lots more to msw outside of this file though. This is just the one that needs to get deployed on its own.

Reply:
Oh I see the version changed, I thought that an edit caused a prettier to run and made a bunch of updates when it said not to modify the file